documentation/modules/kafka-bridge/proc-bridge-deleting-consumer.adoc

```diff
@@ -5,7 +5,8 @@
 [id='proc-bridge-deleting-consumer-{context}']
 = Deleting a Kafka Bridge consumer
 
-Finally, delete the Kafa Bridge consumer that you used throughout this quickstart.
+[role="_additional-resources"]
+Delete the Kafa Bridge consumer that you used throughout this quickstart.
```

Comment:
Here seems to be typo `Kafa` @PaulRMellor

Reply:
@michalxo - Thanks. That will be fixed i https://github.com/strimzi/strimzi-kafka-operator/pull/5827